src/components/Element.js

```diff
@@ -117,7 +121,13 @@ const Element = (type: string, hocOptions: {sourceType?: string} = {}) =>
     };
 
     render() {
-      return <div className={this.props.className} ref={this.handleRef} />;
+      return (
+        <div
+          id={this.props.id || undefined}
```

Comment:
This looks like we're essentially treating `undefined` as the default value of `this.props.id`.

Do you want to change the type of `id` to

```js
type Props = {
  // note the '?' character
  id?: string,
  ...
};
```

and then use `defaultProps` to set `id: undefined`?

Reply:
(Oh hm, we should probably do this with `className` too... up to you whether you want to make either or both changes)

Reply:
I actually tried to do it this way but I think I messed up the proptypes which resulted in an error. Let me go back and do it properly (I normally use typescript so don’t use proptypes so much).